gulpfile.js

```diff
@@ -150,12 +161,12 @@ gulp.task('serve', ['watch'], () => {
 });
 
 gulp.task('build', (cb) => {
-  let args = ['clean', 'assets', 'compile', 'copy', 'scripts', 'styles'];
+  let args = ['clobber-assets', 'copy-assets', 'compile-typescript', 'copy-source', 'browserify', 'preprocess-css', 'validate-yaml'];
```

Comment:
Nit: validate YAML as first step after clean so we fail faster when YAML is incorrect.